README.md

```diff
@@ -80,6 +81,12 @@ format>
 
 Where:
 
+--response-header-with-latency-input <string>
```

Comment:
I think the meaning here is a little hard to parse from the name / comment combo.

How do you feel about response_latency_header_name?